images/macos/macos-10.15-Readme.md

```diff
@@ -69,7 +71,7 @@
 - bazelisk 1.10.1
 - bsdtar 3.3.2 - available by 'tar' alias
 - Curl 7.79.1
-- Git LFS: 2.13.3
+- Git LFS: 3.0.1
```

Comment:
Have we tested this properly?

Reply:
@miketimofeev there was no problem with newest git-lfs on macOS images